sdk/wallet-standard/src/types.ts

```diff
@@ -171,3 +171,8 @@ export type Command =
               data: { [key: string]: unknown };
           };
       };
+
+/**
+ * The first four bytes of the chain's genesis checkpoint digest.
+ */
+export type ChainIdentifier = string;
```

Comment:
What about, instead of basing the solution on the chain identifier `getChainIdentifier,` we base the solution on the chain name using namespaces thats already used by the SDK and the dapp kit? The chain resolution can either live in the dapp or in the SDK, and we already have a chain resolution in the SDK based on `iota:chainName` eg `iota:testnet` (https://docs.iota.org/ts-sdk/api/client/type-aliases/ChainType)
eg
```
"testnet": {
    "id": "testnet",
    "name": "Testnet",
    "url": "https://api.testnet.iota.cafe",
    "explorer": "https://explorer.rebased.iota.org",
    "chain": "iota:testnet",
    "faucet": "https://faucet.testnet.iota.cafe"
}
```